01.get-started/03.installation/03.installer-interactions/docs.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Client and Server Installer and Client Only Installer Interactions
+taxonomy:
+    category:
+        - docs
+---
++ During the client only install, if a Sandbox is still running the install will tell you that one of Sandbox/assignment-client/domain-server is still running and must be closed before install can proceed
++ The client only and server/client installers default to install to different directories, "High Fidelity Interface" and "High Fidelity" respectively.
++ The uninstall record for the client only and client/server installs are the separated in "Apps & Features"
```

Comment:
installs are the separated in "Apps & Features" <-- is the "the" after "installs are" a typo?